skins/oasis/modules/CommentsLikesController.class.php

```diff
@@ -118,38 +112,36 @@ public function executeIndex($data) {
 			$this->commentsEnabled = $this->checkArticleComments();
 
 			// pass accesskey => false to this module to disable accesskey attribute (BugId:15685)
-			if (!isset($data['accesskey']) || $data['accesskey'] !== false) {
+			if ( !isset( $data['accesskey'] ) || $data['accesskey'] !== false ) {
 				$this->commentsAccesskey = ' accesskey="t"';
 			}
 
 			// render comments count as just a bubble
-			$this->commentsBubble = !empty($data['bubble']);
+			$this->commentsBubble = $this->request->getBool( 'bubble' );
 		}
 
-		wfProfileOut(__METHOD__);
+		wfProfileOut( __METHOD__ );
 	}
 
 	public function getData() {
-		global $wgTitle;
-
-		wfProfileIn(__METHOD__);
+		wfProfileIn( __METHOD__ );
 
-		$this->contextTitle = $wgTitle;
+		$this->contextTitle = $this->wg->Title;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)